tests/python/openassetio/hostAPI/test_manager.py

```diff
@@ -514,37 +514,38 @@ def test_when_called_with_parent_then_props_copied_and_createState_called_with_p
         mock_manager_interface.mock.createState.assert_not_called()
 
 
-class Test_Manager_freezeContext:
+class Test_Manager_persistenceTokenForContext:
 
-    def test_when_called_then_the_managers_frozen_token_is_returned(
+    def test_when_called_then_the_managers_persistence_token_is_returned(
```

Comment:
[consider][minor] Not introduced by this PR, but I think these tests would be a candidate for avoiding the `when`/`then` style, since there is no branching logic.

Reply:
Maybe we could pick this up when we get back to #130?